src/SpaceTime/2d/Acceleration2.cs

```diff
@@ -96,20 +84,14 @@ public static Acceleration2 In(Direction2 direction, Acceleration u)
         /// <param name="a0">The acceleration vector at t = 0.</param>
         /// <param name="a1">The acceleration vector at t = 1.</param>
         /// <param name="t">The interpolation scalar.</param>
-        public static Acceleration2 Lerp(Acceleration2 a0, Acceleration2 a1, float t)
-        {
-            return a0 + (a1 - a0) * t;
-        }
+        public static Acceleration2 Lerp(Acceleration2 a0, Acceleration2 a1, float t) => a0 + (a1 - a0) * t;
```

Comment:
This is the only static method here, and I rather have it grouped by functionality, instead of dangling around alone at the bottom.